daq_macros.py

```diff
@@ -24,6 +24,7 @@
 import numpy as np # for runDozorThread
 from string import Template
 from collections import OrderedDict
+from scans import zebraDaqPrep, setupZebraVectorScan, setupZebraVectorScanForRaster
```

Comment:
LSDC is often inconsistent with its naming and daq_macros generally sticks to camel case, but I have done as requested since we expect to enforce consistency (Black/Flake8 formatting) soon.